bus-mapping/src/taiko_anchor.rs

```diff
@@ -0,0 +1,41 @@
+//! The module contains Taiko anchor.
+
+use ethers_core::abi::{Function, Param, ParamType, StateMutability};
+
+use lazy_static::lazy_static;
+
+lazy_static! {
+    static ref ANCHOR_TX: Function = Function {
+        name: "anchor".to_string(),
+        inputs: vec![Param {
+            name: "l1Height".to_string(),
+            kind: ParamType::Uint(256),
+            internal_type: None,
+        },Param {
+            name: "l1Hash".to_string(),
+            kind: ParamType::FixedBytes(32),
+            internal_type: None,
+        }],
+        outputs: vec![],
+        #[allow(deprecated)]
+        constant: Some(false),
+        state_mutability: StateMutability::NonPayable,
+    };
+}
+
+/// Check if a tx is a Taiko anchor tx.
+pub fn is_anchor(tx: &eth_types::Transaction) -> bool {
```

Comment:
you also need to double check this is the 1st tx in the block, maybe you could rename the fn to is_1st_tx_anchor().
Oh, anchor tx has hardcoded from & to, you also need take them into account.

Reply:
Removed.